### PR TITLE
Crosspage dynamic report header added

### PR DESCRIPTION
**Pull request by dima-bond**
image

image

Header for every report with dynamic titles inserted to Layout component.
Minor fixes of responsive design.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,7 +85,7 @@
     "eslint": "7.25.0",
     "eslint-plugin-jest-dom": "3.8.1",
     "eslint-plugin-testing-library": "4.2.0",
-    "husky": "6.0.0",
+    "husky": "^6.0.0",
```
Comment (randomhash) [inline package.json:88]: deps should be installed with flag -E

File: package.json [hunk 2]
```diff
@@ -47,6 +47,7 @@
     "debug": "4.3.1",
     "graphql": "15.5.0",
     "graphql-request": "3.4.0",
+    "husky-run": "^0.0.0",
```
Comment (randomhash) [inline package.json:50]: is it really a dep?
Also -E flag

File: package.json [hunk 3]
```diff
@@ -48,6 +48,7 @@
     "debug": "4.3.1",
     "graphql": "15.5.0",
     "graphql-request": "3.4.0",
+    "husky": "6.0.0",
```
Comment (hmelenok) [inline package.json:51]: Why husky in `dependencies`?

File: src/components/ReportHeader/ReportHeader.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  .ReportHeader {
+    margin: 20px;
+    flex-direction: column;
+    gap: 20px 0;
+
+    &__datepicker {
+      margin-left: 20px;
+    }
+  }
+}
```
Comment (hmelenok) [inline src/components/ReportHeader/ReportHeader.scss:23]: Screenshot 2021-05-20 at 18 57 02

File: src/components/ReportHeader/index.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import ReportHeader from './ReportHeader';
```
Comment (hmelenok) [inline src/components/ReportHeader/index.ts:1]: Don't know what the point of this file?

**Review by hmelenok — APPROVED** [review]

**Pull request merged**